apps/api/src/app/controllers/__tests__/fetchViewModelById.e2e.spec.ts

```diff
@@ -45,20 +47,25 @@ describe('GET /resources (fetch view models)', () => {
     );
 
     beforeAll(async () => {
-        ({ app, arangoConnectionProvider, testRepositoryProvider, commandHandlerService } =
-            await setUpIntegrationTest({
-                ARANGO_DB_NAME: testDatabaseName,
-                BASE_DIGITAL_ASSET_URL: 'https://www.mysound.org/downloads/',
-            }));
+        ({
+            app,
+            arangoConnectionProvider,
+            testRepositoryProvider,
+            idManager,
+            commandHandlerService,
+        } = await setUpIntegrationTest({
+            ARANGO_DB_NAME: testDatabaseName,
+            BASE_DIGITAL_ASSET_URL: 'https://www.mysound.org/downloads/',
+        }));
 
         commandHandlerService.registerHandler(
             'CREATE_SONG',
```

Comment:
We'll probably want a more dynamic way to register all commands or at very least a 'register all commands' test helper.